### PR TITLE
RET- 4165 - Show ECC response notifications + fix status tags

### PR DESCRIPTION
**Pull request by marcjudge**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link
https://tools.hmcts.net/jira/browse/RET-4296
https://tools.hmcts.net/jira/browse/RET-4165
### Change description
image

**Does this PR introduce a breaking change?** (check one with "x")

```
[X] Yes
[ ] No

### CODE REVIEW

**Review by lvo700 — APPROVED** [review]

File: src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts [hunk 1]
```diff
@@ -316,27 +276,41 @@ export const getNotificationResponses = async (
   translations: AnyRecord,
   req: AppRequest
 ): Promise<any> => {
+  const nonAdminRespondCollection = sendNotificationType?.respondCollection
+    ? sendNotificationType.respondCollection
+    : [];
```
Comment (JackReevies) [inline src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts:281]: minor: can be simplified with just `const nonAdminRespondCollection = sendNotificationType?.respondCollection || []`

File: src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts [hunk 2]
```diff
@@ -316,27 +276,41 @@ export const getNotificationResponses = async (
...
+  const adminResponseCollection = sendNotificationType?.respondNotificationTypeCollection
+    ? sendNotificationType?.respondNotificationTypeCollection
+    : [];
```
Comment (JackReevies) [inline src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts:284]: same as above

File: src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts [hunk 3]
```diff
@@ -316,27 +276,41 @@ export const getNotificationResponses = async (
...
-  const respondCollection = sendNotificationType.respondCollection;
-  if (respondCollection?.length) {
-    await addClaimantRespondentResponses(respondCollection, req, translations, allResponses);
-  }
-
-  const adminResponseCollection = sendNotificationType.respondNotificationTypeCollection;
-  if (adminResponseCollection?.length) {
-    await addTribunalResponses(adminResponseCollection, req, allResponses, translations);
+  const combinedResponses = [...nonAdminRespondCollection, ...adminResponseCollection];
+  if (combinedResponses?.length) {
```
Comment (JackReevies) [inline src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts:287]: minor: no need for `?` on combinedResponses

minor: could inverse this condition and return early

**Review by JackReevies — DISMISSED** [review]

File: src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts (hunk 1 above)
Comment (marcjudge) [reply src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts]: Done

File: src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts (hunk 3 above)
Comment (marcjudge) [reply src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts]: Done

**Review by ss-evoco — APPROVED** [review]

**Review by jip1984 — APPROVED** [review]

**Pull request merged**